### PR TITLE
#160530258 ch-mocha-test-fetch-single-order 

### PR DESCRIPTION
**Pull request by Johnpaul-Borree**
#### What does this PR do?
test endpoint GET /api/v1/orders/orderId
#### Description of Task to be completed?
- write tests to fetch single request
- write test to return not found when id is not found
#### How should this be manually tested?
run test locally using npm run test
#### What are the relevant pivotal tracker stories?
#160530258

### CODE REVIEW

File: src/controller/ordersControllers.js [hunk 1]
```diff
@@ -1,37 +1,46 @@
 import Joi from 'joi';
 import orders from '../models/orders';
 
+export function getAllOrders (req, res) {
+	res.status(200).json({status: 'Success', Orders: orders});
```
Comment (KaiserPhemi) [inline src/controller/ordersControllers.js]: #### Critical
Refactor code to make it more readable. Also please follow the Airbnb style guide

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        "quotes": [
+            "error",
+            "single"
+        ],
+        "semi": [
+            "error",
+            "always"
+        ]
+    }
+}
```
Comment (KaiserPhemi) [inline .eslintrc.json]: #### Nitpick 
New line

**Review by KaiserPhemi — CHANGES_REQUESTED** [review]

File: test/ordersTests.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+				.end((err, res) => {
+					res.should.have.status(200);
+					res.body.should.be.a('object');
+					res.body.should.have.property('status').eql('success');
+					res.body.should.have.property('Order').be.a('object');
+					done();
+				});
+		});
+	  });
+});
```
Comment (KaiserPhemi) [inline test/ordersTests.js]: #### Nitpick
New line

File: .eslintrc.json (hunk 2 above)
Comment (Johnpaul-Borree) [reply .eslintrc.json]: just finished implementing this feedback

**Pull request merged**